printer_mods/BlueBear/Bowden_Fence/README.md

```diff
@@ -0,0 +1,14 @@
+# Overview
+
+A little "fence" to hold the tubes away from the belts. Clipmount so no further screws are needed.
+
+![image01](./images/image01.jpg "Usage")
+
+Print one of the clips (either LDO or Makerbeam) together with the BodenFence.stl
+Don't rotate the clips, otherwise they may break upon inserting into the extrusion. They need to be orientated like in the
+picture
+![printorientation](./images/image03.jpg "Print orientation")
+
+Glue the clip into the bodenfence piece and let it cure.
```

Comment:
s/bodenfence/bowdenfence/

Reply:
Sorry, the snap-in must be printed in that orientation otherwise it’s likely to break. Therefore you have to use glue in this mod. I made several approaches with printing the snap-in directly and all broke